korge/src/jvmTest/kotlin/com/soywiz/korge/test/TtfFontTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.soywiz.korge.test
+
+import com.soywiz.korge.testing.*
+import com.soywiz.korge.view.*
+import com.soywiz.korim.font.*
+import com.soywiz.korio.file.std.*
+import org.junit.Test
+
+class TtfFontTest {
+
+    @Test
+    fun disableLigaturesWorks(): Unit = korgeScreenshotTest() {
+        val ttfFontWithLigatures =
+            resourcesVfs["font_atkinson/AtkinsonHyperlegible-Bold.ttf"].readTtfFont()
+        val ttfFontWithoutLigatures =
+            resourcesVfs["font_atkinson/AtkinsonHyperlegible-Bold.ttf"].readTtfFont(enableLigatures = false)
+
+        val t1 = text("with ligatures: 41/41", font = ttfFontWithLigatures, textSize = 40.0) {
+            centerOnStage()
+        }
+        text("without ligatures: 41/41", font = ttfFontWithoutLigatures, textSize = 40.0) {
+            centerOnStage()
+            alignTopToBottomOf(t1)
+        }
+
+        it.recordGolden(this, "text")
+
+        it.endTest()
```

Comment:
I'm a bit worried about this test. Checking the image, it seems that the antialiasing of the text is not working properly. I'm afraid that images with antialiasing or vectors with curves will likely have to be updated while we improve stuff. That PNG has a 14.1KB in size and it is already compressed, so it cannot be compressed in git and deltas won't work because a single small change will produce a totally differenet binary.

Can you disable this test and remove the PNG image, or make it as small as possible? putting in the texts only `1/4`?

Reply:
Done